data/Language/Golang.yaml

```diff
@@ -41,3 +41,27 @@ keywords:
   - switch
   - type
   - var
+datatypes:
+  - chan_go
```

Comment:
this should be `chan` , as the data file for it does not exist.  same with others without a data file.

only add `_go` if the file exists.

Reply:
I understand what you mean, but it will cause conflict if we add data file later. So I thought that way is better.

Reply:
it will not cause a conflict.  when a data file is created for a keyword, the name here will be updated at the same time, from being just the keyword to being the data file name.

Reply:
Ah, thanks :)